spec/customer_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'customer'
+
+describe Customer do
+  
+  subject { Customer.new 075 }
+
+  it 'can request to see a menu' do
+    menu = double('menu', menu_items: {'Chicken' => 5.00, "Salted Beef" => 6.50, "Tuna" => 6.00})
+    expect(subject.request(menu)).to eq (menu.menu_items)
+  end
+
+  it 'can choose items' do
+    dbl = double('choices', choices: ['Chicken', 'Tuna'])
+    expect(subject.choices=dbl).to eq dbl
+  end
+
+  it 'can estimate a total' do
+    expect(subject.total_estimate= 11).to eq 11
```

Comment:
I think you should separate out these lines. Check out the [Four-Phase Test](https://robots.thoughtbot.com/four-phase-test) methodology.